tests/utils/test_geometry_comparisons.py

```diff
@@ -217,6 +219,24 @@ def test_compute_relative_rotation_angle(self):
 
         np.testing.assert_allclose(computed_deg, expected_deg, rtol=1e-3, atol=1e-3)
 
+    def test_compute_relative_rotation_angle_euler(self) -> None:
```

Comment:
Is there any specific bug which we catch from this test? Or does this test help increase coverage?

Reply:
thanks for the review Ayush. was just trying to increase coverage.

At one point I was wondering if there was a bug here, but there wasn't. But I figured i'd add the extra test anyways.

Reply:
Got it. Makes sense!